plugins/meta/flannel/flannel.go

```diff
@@ -92,16 +108,22 @@ func loadFlannelSubnetEnv(fn string) (*subnetEnv, error) {
 			if err != nil {
 				return nil, err
 			}
-			se.mtu = uint(mtu)
+			se.mtu = new(uint)
```

Comment:
oh ok clearly I didn't read the spec of ParseUint. *cringe* sometimes I miss Java's extreme type verbosity..